python/pyspark/pandas/data_type_ops/num_ops.py

```diff
@@ -110,13 +126,18 @@ def pow(self, left: IndexOpsLike, right: Any) -> SeriesOrIndex:
         if not is_valid_operand_for_numeric_arithmetic(right):
             raise TypeError("Exponentiation can not be applied to given types.")
 
-        Column = ConnectColumn if is_remote() else PySparkColumn
+        if is_remote():
+            from pyspark.sql.connect.column import Column as ConnectColumn
 
-        def pow_func(left: GenericColumn, right: Any) -> GenericColumn:
+            Column = ConnectColumn
+        else:
+            Column = PySparkColumn  # type: ignore[assignment]
+
+        def pow_func(left: Column, right: Any) -> Column:  # type: ignore[valid-type]
```

Comment:
ditto

Reply:
This is exception since `Column` is used in function.